### ISSUE TITLE
Ethereum Bridge (5000 Gold)

### ISSUE THREAD

**Issue by mikera** [Convex-Dev/bounties#11]
The project is to build a working Bridge that can connect a Convex network (e.g. the Testnet) to an Ethereum Network.

Objective:
- Must be possible to run on a secure server which also operates as a Convex Peer
- Must allow ERC20 / ERC777 and similar tokens to be transferred to and from Convex as a wrapped fungible token
- Wrapped assets on Convex must conform to the standard `convex.asset` interface
- Must be demonstrated linking the Convex mainnet (once launched) to the Ethereum mainnet

The Bridge itself need not be decentralised, but must support fully decentralised networks on both the Convex and Ethereum side

**Comment by pferdy** [comment]
Could a bridge be general purpose?  In other words, could it be awarded if demonstrated with any other wrapped fungible tokens?  If so it could be developed as part of NGI TrustChain OC4 for Interoperability.

**Comment by mikera** [comment]
Yes, it could be general purpose. This specific bounty is for an Ethereum bridge, but a similar award could be made for a generalised Bridge.
